src/runtime/lpython_builtin.py

```diff
@@ -963,6 +963,8 @@ def _lpython_str_isascii(s: str) -> bool:
     return True
 
 def _lpython_str_isspace(s:str) -> bool:
+    if (len(s) == 0):
+        return False
```

Comment:
Looks good. We could skip the brackets though, `if len(s) == 0` should do !
Also add a test for the same.